### PR TITLE
added filter by path and tag to unaired and nextaired methods

### PR DESCRIPTION
**Pull request by redglory**
Following the change in next episodes which can be filtered by path and tag parameters, I've applied same concept to nextairing and unaired episodes, which relie on the script.module.thetvdb module.

Also added pt_pt full translation .po

### CODE REVIEW

File: resources/lib/episodes.py [hunk 1]
```diff
@@ -203,13 +202,39 @@ def get_next_episode_for_show(self, show_id):
...
+        if self.options.get("tag"):
+            filters.append({"operator": "contains", "field": "tag", "value": self.options["tag"]})
+        if self.options.get("path"):
+            filters.append({"operator": "startswith", "field": "path", "value": self.options["path"]})
+
+        # First we get a list of all the inprogress/unwatched TV shows ordered by lastplayed
+        all_shows = self.metadatautils.kodidb.tvshows(sort=kodi_constants.SORT_LASTPLAYED, filters=filters,
+                                                      limits=(0, self.options["limit"]))
+        tvshows_ids = [d['tvshowid'] for d in all_shows]
+        episodes = self.metadatautils.thetvdb.get_kodi_unaired_episodes(True, tvshows_ids)
```
Comment (marcelveldt) [inline resources/lib/episodes.py]: please change this line in 

episodes = self.metadatautils.thetvdb.get_kodi_unaired_episodes(True, False, tvshows_ids)

see my comments in the PR for tvdb module

File: resources/lib/episodes.py [hunk 2]
```diff
@@ -203,13 +202,39 @@ def get_next_episode_for_show(self, show_id):
...
+        if self.options.get("tag"):
+            filters.append({"operator": "contains", "field": "tag", "value": self.options["tag"]})
+        if self.options.get("path"):
+            filters.append({"operator": "startswith", "field": "path", "value": self.options["path"]})
+
+        # First we get a list of all the inprogress/unwatched TV shows ordered by lastplayed
+        all_shows = self.metadatautils.kodidb.tvshows(sort=kodi_constants.SORT_LASTPLAYED, filters=filters,
+                                                      limits=(0, self.options["limit"]))
+        tvshows_ids = [d['tvshowid'] for d in all_shows]
+        episodes = self.metadatautils.thetvdb.get_kodi_unaired_episodes(False, tvshows_ids)[:self.options["limit"]]
```
Comment (marcelveldt) [inline resources/lib/episodes.py]: please change this line in

episodes = self.metadatautils.thetvdb.get_kodi_unaired_episodes(False, False, tvshows_ids)
episodes = episodes[:self.options["limit"]]

see my comments in the PR for tvdb module

**Review by marcelveldt — CHANGES_REQUESTED** [review]
thanks for the PR, please see my comments in both PR's (also in tvdb module) before I can accept the merge

**Comment by redglory** [comment]
You are right!

Fixed thetvdb tvshows_ids parameter and made it optional.

**Comment by marcelveldt** [comment]
Thanks !

**Pull request merged**